src/core/template-wrapper.ts

```diff
@@ -13,9 +13,19 @@ interface ITemplateWrapperProps {
 }
 
 class TemplateWrapper extends React.PureComponent<ITemplateWrapperProps> {
+    private _removeListener: HTMLDivElement;
```

Comment:
Sounds like it is a function, what about renaming to `_removalListener`  or `_eventsListener`?